git_machete/github.py

```diff
@@ -296,7 +295,7 @@ def __fire_github_api_request(self,
                 else:
                     raise MacheteException(
                         first_line + f'You might not have the required permissions for this repository.\n'
-                                     f'Provide a GitHub API token with `repo` access via {self.__token.provider}.\n'
+                                     f'Provide a GitHub API token with `repo` access.\n'
```

Comment:
Logical error, we wanted to print token provider in the else statement which is only entered when token is None

Reply:
Whoops ☹️